README.md

```diff
@@ -20,53 +20,40 @@ Script to test the concurrent connections towards a server listening to a TCP po
 Usage of ./tcpMaxConn:
   -connections int
         Number of connections you want to open (default 100)
+  -debug
+        Print debugging information to standard error
   -delay int
         Number of ms you want to sleep between each connection creation (default 10)
   -host string
         Host you want to open tcp connections against (default "localhost")
+  -interval int
```

Comment:
good point, I also reached the same point yesterday and I would suggest it 👍

Reply:
i really think this is something outside the scope of this PR/issue. I've open this to discuss:
https://github.com/dachad/check-max-tcp-connections/issues/13

Reply:
👍